docs/en_us/platform_api/source/enrollment/overview.rst

```diff
@@ -0,0 +1,37 @@
+.. _edX Enrollment API Overview:
+
+################################################
+EdX Platform Enrollment API Overview
+################################################
+
+Use the edX Platform Enrollment API to view user and course enrollment
+information, and to enroll a user in a course.
+
+The edX Platform Enrollment API uses Representational State Transfer (REST)
+design principles and supports the JavaScript Object Notation (JSON) data-interchange format. The REST API is simple, lightweight, and optimized.
```

Comment:
Nitpick: I'm not even sure what optimized would mean in this context. Can we please remove it?